### PR TITLE
[READY] Removes the Genetics Lab and the Geneticist job and makes it require advanced biotech.

### PR DESCRIPTION
**Pull request by GrayRachnid**
## About The Pull Request
So, with MDs being given access to the cloning lab, the geneticist job has become moot other than fishing for powers and then lazing off. The issue isn't there however, it's how often genetics has been used/abused as of late repeatedly. People who know how genetics work would rush the superpowers they want and have an advantage over antagonists. Genetics in its current state is nothing but a free powergaming tool, upgrades make it faster yes but you can still unlock some power with an un-upgraded genetics.

The cloning lab is not being touched, it's staying there. The genetics lab will be made some kind of abandoned genetics lab though, i'll work the map out after I get some sleep, this is just the code work.

## Why It's Good For The Game
Removes an unnecessary job and hopefully reduces the amount of roundstart powergaming. It's okay to build your own genetics lab later into the round and unlocking powers but it shouldn't be ok for an assistant/security to hack into genetics, unlock powers, and have an advantage over other antagonists for the rest of the round.

## Changelog
:cl:
del: Removed geneticists as a job and a lab.
balance: Genetics equipment is now locked behind advanced biotech rather than just biotech.
config: Removed genetics from config.
/:cl:

### CODE REVIEW

**Review by silicons — DISMISSED** [review]
EPIC

**Comment by BlackMajor** [comment]
Local man with the thermonuclear nerf.

(Kevinz this is what you get for taking forever to nerf genetics yourself)

**Comment by chefferz** [comment]
Entirely against this ham fisted approach. Genetics has been around since 2014, and just because people break in and use it, those people should be dealt with rather than this kind of reaction. 

I want hard statistics on how often it's abused, too. This isn't a frequent thing. I'd really say only 20% or so rounds even have an active, competent geneticist on, and that's pushing it. 

You want a suggested balance? Here's some from someone who actually has hundreds if not thousands of hours in genetics.

1) Set the xray power to only manifest on a F** block, or even FD*if you really wanna delay it, because getting that would only be pure luck unless you got higher tier parts.
2) Fully remove science access from genetics, and genetics access from science. Bam, now it's considered powergaming and you can easily ahelp if people truly break in and "abuse" it.
3) Set the machines to require better parts for more consistent mutations, so it's delayed a bit.

[https://i.imgur.com/ouKpmFv.png](url) < Unacceptable.

**Comment by deathride58** [comment]
Yeah no, this here is a hard, *hard* no unless you can give a damn good reason to not go with the obvious solution of maybe nerfing it instead.

**Comment by Thalpy** [comment]
Genetics is on my to do list, after Christmas map, Organ traumas, Tamiorgans, Chem jiggles and Polymers.

**Comment by dapnee** [comment]
Oh cool, another thing for science to do. And yes, I know RD’s can and will do genetics, but this is completely removing it from medbay. I still feel a straight removal PR is trash and would like alternate means focused on before this is even considered.

**Comment by RogueSteampunker** [comment]
Yeah, gonna have to go ahead and give a hard yikes to this one. The issue isn't genetics existing to begin with, it's the fact that powers are overly strong. Why we'd remove all of genetics, and remove the ability to change appearances and the like without rnd researching an out-of-the-way node is beyond me.

**Comment by GrayRachnid** [comment]
> 
> 
> Yeah no, this here is a hard, _hard_ no unless you can give a damn good reason to not go with the obvious solution of maybe nerfing it instead.

Geneticists as of right now have a pointless role. Their whole job is to rush and unlock superpowers that when compared to other aspects of the game, are way too easy to obtain with absolutely no downsides. A comparison point is augments, augments are locked behind much much further tech nodes and REQUIRE another person to install them into you, plus their weakness is EMPs. Genetics can be done by a single person and they'd have up to 5 powers that gives them an advantage over other players/antagonists. This wasn't an issue before, and neither is it an issue on other servers that still use this genetics system for two reasons. Before, we didn't have many people using it as much as it's being rushed these days(at least one person rushes genetics every other round, and during late nights it is rushed every single round by a few different people) and antags weren't always fighting an uphill battle. Citadel in its current state gives solo antagonists a huge disadvantage. Citadel doesn't have the chaos and unrulyness of other servers that is caused by greytiders and the like, plus antags are discouraged from murderboning. This makes validhunting one of the few ways to kill boredom and gain some gratification from making spessmen horizontal. That is an issue, a much deeper issue than a simple Genetics removal PR but this simply aids in giving antagonists less of a disadvantage when the whole crew is chasing them down for valids. Baby steps. And as I forgot to mention in my opener, Genetics as a job is wholly pointless right now. Their whole duty is 'get powers and do nothing' due to the fact that the cloning lab can now be accessed by MDs. Chemists have also become competent, so you no longer need to setup clean SEs, 80% of the time there's a mutadon, mannitol, cryo mix in cryo for early pre-upgrades cloning. MDs have basically taken over the Cloning lab and Genetics no longer has a role in medbay, their only purpose now is science research and that boils down to superpowers which as discussed above, are unfair and overpowered. And, this is the /nerf/ to Genetics. It's a nerf because it delays genetics until after the bomb is complete rather than it being rushed shift-start. Genetics machines are also locked to the Engineering Circuitprinter and Medbay Fabricator Lathe, keeping Science out of it. This way, Medbay is the one who ultimately owns Genetics even though they have to ask for parts from science. This is nerfing genetics except with extra steps. The extra steps of removing the genetics role is due to how useless it has become over the years. Yes, there are other ways to nerf genetics. Yes, I could just ahelp the powergamers. But here's my counterpoint to both these points.

1. My first nerf to genetics would be removing Xray, not making it flyperson only, not replacing it with something like thermals. No, full blown removal. Xray is the main culprit of this PR due to how overpowered it is and how much information it gives the player that is otherwise unattainable. A superpower that gives you so much information with absolutely no downsides is overpowered. Hands down infact one of the most overpowered abilities in a game centered around distrust and secrecy. Frankly, XRay as a superpower is a novel idea and I don't want to see it removed, no amount of balancing will fix it, geneticists will just fly themselves the same way they do to unlock hulk, thermals is no better. The only way in my opinion to nerf XRay is that it gives you brain damage for every tick that you have it in for(due to the overload of information) the damage also increasing the longer you have it, at a certain point you are no longer able to have xray(removing it and re-adding it would not reset this), which might be even more contested than removing geneticist as a role.
2. Admins are not omnipotent. This is true, you should ahelp every situation you come across if you think it breaks the rules. However, admins are also not blind. This has been going on for weeks and that's only what i've seen but apparently it's also been happening for months. If such a glaring powergaming recurring issue has been happening for months with nobody discouraging it or dishing out bans, then clearly it's not a rule issue but a balance issue, so here we are.

Bhijn, i've tried to give as many reasons as I could possibly think of, if none of these convince you then give me an alternative. Also, Ana, that message about locking genetics behind more nodes was as much as a joke as Kevinz insta approving my PR, it wasn't serious.

File: code/game/objects/effects/landmarks.dm [hunk 1]
```diff
@@ -167,10 +167,6 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 	name = "Research Director"
 	icon_state = "Research Director"
 
-/obj/effect/landmark/start/geneticist
```
Comment (Ghommie) [inline code/game/objects/effects/landmarks.dm:170]: Remember to edit the station maps .dmms.

**Review by Ghommie — COMMENTED** [review]
Oh well, nobody ever bothered improving genetics here anyway.

File: code/game/objects/effects/landmarks.dm (hunk 1 above)
Comment (GrayRachnid) [reply code/game/objects/effects/landmarks.dm:170]: dun, I did this pr at a late time and i left map changes for when i woke up

**Comment by Thalpy** [comment]
Make mutadone give temporary relief, and then genetics has a role again.

**Comment by GrayRachnid** [comment]
> 
> 
> Make mutadone give temporary relief, and then genetics has a role again.

This solves one issue but doesn't resolve the issue of genetics being overpowered.

**Comment by GrayRachnid** [comment]
> 
> 
> Yeah, gonna have to go ahead and give a hard yikes to this one. The issue isn't genetics existing to begin with, it's the fact that powers are overly strong. Why we'd remove all of genetics, and remove the ability to change appearances and the like without rnd researching an out-of-the-way node is beyond me.

You can save your UI/UE on a genetics diskette using the cloning lab scanner if you're afraid of losing how you look, otherwise you will unfortunately have to wait until medbay builds genetics machines.

**Comment by Thalpy** [comment]
Have you considered using hyper's system? Since they're downstream it should be easy.

**Comment by GrayRachnid** [comment]
> 
> 
> Have you considered using hyper's system? Since they're downstream it should be easy.

Both kevinz and bhijn have deneid it.

**Comment by Trilbyspaceclone** [comment]
This ruins peacefull/stealth revs monkey static :(

**Comment by ghost** [comment]
someone died to a genetcist lol

**Comment by GrayRachnid** [comment]
> 
> 
> someone died to a genetcist lol

I wish it was a geneticist.

**Comment by YakumoChen** [comment]
Goon genetics have been denied only because it uses laggy tgui, not because it isnt a good system.

**Comment by Tename** [comment]
I always had fun playing genetics and making a nice little collection of every negative effect and positive effect, it would be very depressing to see this get merged and turn into another science gimmick thing.

**Comment by silicons** [comment]
i denied goon genetics because i shouldn't need 50 ms ping to properly play a job without abusing the shit out of the handheld scanners.

bhijn denied it for another reason.

if someone can resolve the former, potentially by making it so it's buttons/tgui/byond map based interface rather than fancy html interface i'd be fine

yes, even making it tgui + 5 buttons for each block would p r o b a b l y make the lag better, the issue is the input() call from you clicking it and it going to Topic(), you putting it in, it going back to the proc, and the view updating is way, way too much and just amplifies the shit out of your actual ping.

**Comment by xurxomario** [comment]
>A comparison point is augments, augments are locked behind much much further tech nodes and REQUIRE another person to install them into you, plus their weakness is EMPs.

Reminder that a singular mutadone syringe in a syringe gun will instantly remove any and all mutations in a single second, which makes genetics powers one of the easiest stuff to counter, no matter whether you or the geneticist is an antag.

Also, hard no, as has been stated, dont remove features, rework or nerf em, removal should never be the way, and the bombcap is living proof of it.

**Comment by GrayRachnid** [comment]
>Also, hard no, as has been stated, dont remove features, rework or nerf em, removal should never be the way, and the bombcap is living proof of it.

This isn't removing any features, this is removing a pointless role and making genetics a lategame technology only accessed by medbay.

**Comment by Thalpy** [comment]
Isn't the removal of the genetics job a feature removal? 

If you want, help me finish my current projects and I'll take a crack at it.

**Comment by dapnee** [comment]
You can still research the machines, if anything, it’s a lazy way to limit X-ray eyes by like, 30min-1hour in game.

**Comment by ShadeAware** [comment]
All this will do is make Security hound Science to build a genetics lab for X-ray or, at worst, waste points on said lab near roundstart to do it themselves.

**Comment by GrayRachnid** [comment]
As stated previously, as of right now Geneticists do nothing to contribute to medbay or science. They are just there to unlock super powers which with our current iteration of genetics, is very easy. This removes the geneticist role which doesn't contribute to the station other than for powergaming and makes their lab a Technode that only Medbay is able to print circuits of.

**Comment by Thalpy** [comment]
Well, I was hoping to make the geneticist the means of curing genetic organ traumas

**Comment by GrayRachnid** [comment]
> 
> 
> Well, I was hoping to make the geneticist the means of curing genetic organ traumas

Ah, I wasn't aware of that. I'm discontinuing this PR just for that then.

**Comment by Thalpy** [comment]
Heck, why don't we work together to change genetics? I'm happy to code organ traumas with you.

**Comment by GrayRachnid** [comment]
> 
> 
> Heck, why don't we work together to change genetics? I'm happy to code organ traumas with you.

sure but am bad at coding

**Comment by Thalpy** [comment]
The difference between a good coder and a bad coder is just the willingness to put in effort! So you must be a good coder being coy!

**Comment by GrayRachnid** [comment]
i'm just kidding this was made in response to the hulk species pr

**Pull request closed without merging**